### PR TITLE
KEP-3325: Promote SelfSubjectReview to Beta

### PR DESCRIPTION
**Pull request by nabokihms**
Documentation for:

* KEP (k/enhancements) update PR(s): https://github.com/kubernetes/enhancements/pull/3772
* Code (k/k) update PR(s): https://github.com/kubernetes/kubernetes/pull/116274
* Docs (k/website) update PR(s): https://github.com/kubernetes/website/pull/39794

### CODE REVIEW

**Review by sftim — COMMENTED** [review]
I'm afraid we can't accept the unrelated whitespace changes. Those will cause merge conflicts or force people to rebase when working on other changes.

You'll need to fix `content/en/docs/reference/command-line-tools-reference/feature-gates.md` @nabokihms so there are only a few lines that change.

**Comment by sftim** [comment]
/hold
until concern in https://github.com/kubernetes/website/pull/39794#pullrequestreview-1325800418 is addressed

**Comment by nabokihms** [comment]
@sftim, sorry for this. My IDE autoformatted it.

**Comment by nabokihms** [comment]
@sftim the code PR is merged. Do you need anything else to unhold this PR?

File: content/en/docs/reference/command-line-tools-reference/feature-gates.md [hunk 1]
```diff
@@ -63,6 +63,7 @@ For a reference to old feature gates that are removed, please refer to
 | `APIResponseCompression` | `false` | Alpha | 1.7 | 1.15 |
 | `APIResponseCompression` | `true` | Beta | 1.16 | |
 | `APISelfSubjectReview` | `false` | Alpha | 1.26 | |
```
Comment (tengqm) [inline content/en/docs/reference/command-line-tools-reference/feature-gates.md:65]:
```suggestion
| `APISelfSubjectReview` | `false` | Alpha | 1.26 | 1.26 |
```

**Comment by liggitt** [comment]
/retest
/hold cancel
technical content lgtm (for sig-auth)

File: content/en/docs/reference/access-authn-authz/authentication.md
Comment (sftim) [inline content/en/docs/reference/access-authn-authz/authentication.md:1]: BTW, is ``kubectl alpha auth whoami` still the right command to use?

Comment (liggitt) [reply content/en/docs/reference/access-authn-authz/authentication.md:1]: no, the command moved to `kubectl auth whoami` in https://github.com/kubernetes/kubernetes/pull/116510

**Comment by sftim** [comment]
@nabokihms please consider updating the `kubectl auth whoami` command as well; it's strongly related.

**Comment by nabokihms** [comment]
I will take a look today.

**Pull request merged**